src/app/models/types/curationData.ts

```diff
@@ -56,6 +56,9 @@ export interface BaseCuration {
   curationType?: string;
   mostRead?: MostReadData;
   radioSchedule?: RadioScheduleData[];
+  embed?: {
+    html: string;
+  };
```

Comment:
Do we maybe want to use the `OEmbedData` type from https://github.com/bbc/simorgh/blob/e2abf581b77d21d9309720e348f7fee2e85416b8/src/app/components/Embeds/types.ts#L9?